lib/audio_output/src/DACOutput.cpp

```diff
@@ -2,15 +2,15 @@
 
 #if CONFIG_IDF_TARGET_ESP32 || CONFIG_IDF_TARGET_ESP32S2 || CONFIG_IDF_TARGET_ESP32S3
 
-void DACOutput::start(int sample_rate)
+void DACOutput::start(unsigned int sample_rate)
 {
     // i2s config for writing both channels of I2S
     i2s_config_t i2s_config = {
         .mode = (i2s_mode_t)(I2S_MODE_MASTER | I2S_MODE_TX | I2S_MODE_DAC_BUILT_IN),
         .sample_rate = sample_rate,
         .bits_per_sample = I2S_BITS_PER_SAMPLE_16BIT,
         .channel_format = I2S_CHANNEL_FMT_RIGHT_LEFT,
-        .communication_format = (i2s_comm_format_t)(I2S_COMM_FORMAT_I2S_MSB),
+        .communication_format = I2S_COMM_FORMAT_STAND_I2S,
```

Comment:
I2S_COMM_FORMAT_I2S is deprecated, correspond to I2S_COMM_FORMAT_STAND_I2S

Reply:
I'll update the code to the IDF 4.4.x i2s API (Arduino ESP32 core 2.0.8) in the next days, but I still want to be compatible with ESP32 core 1.0.6 too, so I will add some ifdef´s. Also I´d like to test on S2/S3 and fix the compilation issues.